src/threading/MsgThread.cc

```diff
@@ -227,6 +225,83 @@ MsgThread::MsgThread() : BasicThread(), queue_in(this, nullptr), queue_out(nullp
 
     io_source = new detail::IOSource(this);
 
+    cnt_sent_in_metric = telemetry_mgr->CounterInstance("zeek", "msg_thread_msgs_sent_in", {{"thread_name", Name()}},
+                                                        "Number of messages sent into thread");
+    cnt_sent_out_metric = telemetry_mgr->CounterInstance("zeek", "msg_thread_msgs_sent_out", {{"thread_name", Name()}},
```

Comment:
These two names where he before, but I don't think we need Hungarian notation here (i.e., drop the `cnt_` prefixes). Ideally we'd have consistent naming, see my previous comments.

Also check below for consistent naming.